link_resolver.go

```diff
@@ -138,9 +167,10 @@ func doRequest(urlString string, r *http.Request) (interface{}, error, time.Dura
 	}
 
 	response := &LinkResolverResponse{
-		Status:  resp.StatusCode,
-		Tooltip: tooltip.String(),
-		Link:    resp.Request.URL.String(),
+		Status:    resp.StatusCode,
+		Tooltip:   tooltip.String(),
+		Link:      resp.Request.URL.String(),
+		Thumbnail: data.ImageSrc,
```

Comment:
The thumbnail here needs to be proxied the same way we do with direct image links, see the code block right below